src/AElf.WebApp.Application.Chain/AppServices/AppTransactionResultService/AppTransactionGetResultService.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using AElf.Kernel.Blockchain.Application;
+using AElf.Kernel.Blockchain.Domain;
+using AElf.Kernel.TransactionPool.Infrastructure;
+using AElf.Types;
+using AElf.WebApp.Application.Chain.Dto;
+using Microsoft.AspNetCore.Routing.Template;
+using Newtonsoft.Json;
+using Volo.Abp.Application.Services;
+using Volo.Abp.DependencyInjection;
+
+namespace AElf.WebApp.Application.Chain.AppServices.AppTransactionResultService
```

Comment:
do not create new namespace, just put under AElf.WebApp.Application.Chain

Reply:
Roger that ~